### ISSUE TITLE
[Problem] UI dies while scanning PS1 discs for anti-mod strings

### ISSUE THREAD

**Issue by Kreeblah** [SabreTools/MPF#265]
**Before You Submit**
- Remember to try the [latest WIP build](https://ci.appveyor.com/project/mnadareski/mpf/build/artifacts) to see if the issue has already been addressed.
- Is it copy protection related? If so, report the issue [here](https://github.com/mnadareski/BurnOutSharp/issues) instead.
- .NET Core 3.1 and .NET 5.0 have known issues, please try using another build to reproduce the error
- Check multiple discs to help narrow down the issue
- Check the Options to see if changing any of those affects your issue.

If all of those fail, then continue...

**Version**
What version are you using? 

- [ ] Stable release (version here)
- [x] WIP release https://github.com/SabreTools/MPF/commit/bfbae5cf5c9d1d406eac77fcb162c8acd8dfd355

**Build**
What runtime version are you using?

- [ ] .NET Framework 4.7.2 running on (Operating System)
- [x] .NET Framework 4.8 running on Windows 10
- [ ] .NET Core 3.1 running on (Operating System)

**Describe the issue**
I've only tried this with two PS1 discs (Descent and Descent Maximum) so far, but it's happened with both.  While scanning for anti-mod strings, the UI suddenly quits.  There is still a DiscImageCreator.exe process which continues running, and locks the files in the output directory, but I'm not really sure what it's doing.

This seems to happen around the 65000-75000 sector range, and it happens every time I try to read one of these discs.

**To Reproduce**
1. Configure the app to use the CLI programs from the 1.18 bundle (in case it makes a difference)
2. Insert either Descent for PS1 or Descent Maximum for PS1 (or possibly other PS1 discs)
3. Make sure it was detected as a Sony Playstation disc
4. Begin dumping
5. Make sure it starts scanning for anti-mod strings
6. Wait for the UI to quit

**Expected behavior**
I'd expect the UI to not quit.

**Screenshots**

**Additional context**
I haven't seen anything in the output files that looks super relevant here, but I might have missed something.

**Comment by Kreeblah** [comment]
Oh, I'm also not sure whether it's related to BurnOutSharp or not.  If it is, I can report the issue there instead.

**Comment by mnadareski** [comment]
Since it's happening before the dump as a part of DIC, it's something about DIC's outputs at that point that is causing the issue. I'm assuming that DIC's outputs are going to the Log output instead of a separate window in your case?

**Comment by Kreeblah** [comment]
They are, yeah.

**Comment by mnadareski** [comment]
Okay, I did try to reproduce this locally to no avail. I then added https://github.com/SabreTools/MPF/commit/4842bdd38b93bfc69ba810e7dbce7c78e8bf77b4 to ensure that if there's a logging error (like during processing of DIC outputs) it will just log the error and continue.

Even with that (and an additional change to process the anti-mod search https://github.com/SabreTools/MPF/commit/2d19d51bc7a14a33560148369c0bae86f858d4df) I still haven't been able to observe the same results.

**Comment by Kreeblah** [comment]
Well, I just tried https://github.com/SabreTools/MPF/commit/2d19d51bc7a14a33560148369c0bae86f858d4df and I can't reproduce the issue at all.  With https://github.com/SabreTools/MPF/commit/bfbae5cf5c9d1d406eac77fcb162c8acd8dfd355 it happened 100% of the time, but I can't get it to happen with this commit.  So, I think it's good.

I did notice one cosmetic thing, though, with the anti-mod progress bar.  It says "anit-mod" instead of "anti-mod".  Anyway, I think this can probably be closed.  Thanks for taking a look at it!
